samples/FakeItEasy.Examples.ruleset

```diff
@@ -15,4 +14,4 @@
     <Rule Id="CA2201" Action="None" />
     <Rule Id="CA2210" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
AFAIK editorconfig only applies when you're manipulating the raw text file itself (in VS or any other editor which supports editorconfig). If you use the "ruleset GUI" in VS then it bypasses editorconfig since, I'm guessing, the editorconfig extension doesn't get the required (saving a text file) events. It's probably easier to alter our editorconfig settings to stop trying to add EOL's at the end of file types that battle against the various GUI's in VS for various file types. I believe we've already done this for csproj and vbproj.